metrics-exporter/src/bin/io_engine/cache/mod.rs

```diff
@@ -1,14 +1,13 @@
 mod pool;
+mod pool_stat;
 
-use crate::{
-    client::{grpc_client::GrpcClient, pool::Pools},
-    ExporterConfig,
-};
+use crate::client::{grpc_client::GrpcClient, pool::Pools};
```

Comment:
doesn't seem linted

Reply:
i tried to have them in new line. But after running linter it puts it back on same line.

Reply:
that's my point, it should be on the same line because the base import is the same, that's why the linter is doing it :)